frontend/public/components/nav/admin-nav.tsx

```diff
@@ -161,7 +162,7 @@ const AdminNav = () => (
       <ResourceNSLink resource="rolebindings" name="Role Bindings" startsWith={rolebindingsStartsWith} />
       <ResourceNSLink resource="resourcequotas" name="Resource Quotas" startsWith={quotaStartsWith} />
       <ResourceNSLink resource="limitranges" name="Limit Ranges" />
-      <ResourceNSLink resource={referenceForModel(ChargebackReportModel)} name="Chargeback" required={FLAGS.CHARGEBACK} />
+      <HrefLink href={formatNamespacedRouteForResource(referenceForModel(ChargebackReportModel), 'openshift-metering')} name="Chargeback" required={[FLAGS.CHARGEBACK, FLAGS.CAN_LIST_CHARGEBACK_REPORTS]} />
```

Comment:
Do we need to specify `startsWith` here?

Reply:
Added `startsWith`

Reply:
Added `startsWith`

Reply:
When I drilldown into a Reports detail page, the url changes, and 'Chargeback' in the nav menu remains highlighted/selected, so I don't think we need `startsWith` in this case.